vaccine_feed_ingest/schema/schema.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+
+class Address(BaseModel):
+    """
+    {
+        "street1": str,
+        "street2": str,
+        "city": str,
+        "state": str as state initial e.g. CA,
+        "zip": str,
+    },
+    """
+
+    street1: str
+    street2: Optional[str]
+    city: str
+    state: str
+    zip: str
+
+
+class LatLng(BaseModel):
+    """
+    {
+        "latitude": float,
+        "longitude": float,
+    },
+    """
+
+    latitude: float
+    longitude: float
+
+
+class Contact(BaseModel):
+    """
+    {
+        "contact_type": str as contact type enum e.g. booking,
+        "phone": str as (###) ###-###,
+        "website": str,
+        "email": str,
+        "other": str,
+    }
+    """
+
+    contact_type: Optional[str]
+    phone: Optional[str]
+    website: Optional[str]
+    email: Optional[str]
+    other: Optional[str]
+
+
+class OpenDate(BaseModel):
+    """
+    {
+        "opens": str as iso8601 date,
+        "closes": str as iso8601 date,
+    }
+    """
+
+    opens: Optional[str]
+    closes: Optional[str]
+
+
+class OpenHour(BaseModel):
+    """
+    {
+        "day": str as day of week enum e.g. monday,
```

Comment:
To the extent possible, I would _really_ like to have business hours in a structured form (so that we can direct phone bankers/call centres to open locations, especially given time zones). I am okay with deciding that the normalization happens somewhere else, but it does need to live somewhere.

Reply:
At least at the vial level, we need to be able to ask the question "is this open now"? 

We won't have the data everywhere, but we should be representing it as spans of time that the location is open. The spec @bryanculbertson put together had this normalized to an array of "day of week, opens at, closes at" tuples. This neatly solves the problem of a pharmacy being open from 9-noon and then 1-6 on sundays.  

At the same time, we get a lot of really bad opening hours data and it may make sense to have an escape hatch to include an unstructured "opening hours in prose" field of some kind, so we don't have to implement all of RFC2445's recurrence calculation engine?

I'd much rather we be able to get data into the database and then iterate on parsing out opening hours later.

Reply:
Per discussion on Discord:
* Current schema is based on https://schema.org/OpeningHoursSpecification.
* May also want to track "hard-to-normalize free text hours," but this will be added later if desired.

Leaving as-is.